docs/hello.rst

```diff
@@ -3,7 +3,14 @@ Hello, world!
 
 We introduce the Taichi programming language through a very basic `fractal` example.
 
-Running the Taichi code below (``python3 fractal.py``) will give you an animation of
+.. note::
+
+    A successful installation of Taichi should add a command-line interfce to your system.
+    You could use it to run bundled examples for convenience. Please run ``ti example -h``
+    for more information.
+
+
+Running the Taichi code below (``python3 fractal.py`` or ``ti example fractal``) will give you an animation of
```

Comment:
```suggestion
Running the Taichi code below (``python3 fractal.py``) will give you an animation of
```
IMHO this may confuse people.
Currently `ti example fractal` doesn't show the source code of `fractal`.
And user can't edit `fractal.py` on their own for study attempt.

So I think we should add another section in `Miscellaneous`, called `user_utilities.rst`, which contains end-user utilities:
1. `ti example`
2. `ti changelog`
3. `ti gif`
4. `ti video` (#1116)
5. how to play with the built-in profiler: `ti.profiler_start/stop/print`
6. how play with out of bound check by `ti.init(debug=True)`

Or how about rename `utilities.rst` to `dev_utilities.rst`, and `user_utilities.rst` to `utilities.rst`?

Reply:
> Currently ti example fractal doesn't show the source code of fractal.
And user can't edit fractal.py on their own for study attempt.

Thanks, that makes sense to me! Yeah, it'd be better to avoid any potential complication/confusion in the hello.rst.

> Or how about rename utilities.rst to dev_utilities.rst, and user_utilities.rst to utilities.rst?

This sounds great to me. Let me go down with this path! (Since #1116 is still ongoing, I will keep the scope of this PR to simply introduce `ti example` in `utilities.rst` and we could re-organize the other content you mentioned above altogether, wdyt?)